src/libraries/System.Net.Http/src/Resources/Strings.Designer.cs

```diff
@@ -0,0 +1,925 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
```

Comment:
Not sure how this file got added, but it shouldn't be. It's autogenerated during the build.

Reply:
removed